test/functional/connection/connection.ts

```diff
@@ -359,5 +359,31 @@ describe("Connection", () => {
         })));
 
     });
+
+    describe("Can change postgres default schema name", () => {
+        let connection: Connection;
+        afterEach(() => connection.isConnected ? connection.close() : {});        
+        it("schema name can be set", async () => {
+            const testSchema = "test-schema";
+            const testOption = setupSingleTestingConnection("postgres", {
+                name: "default",
+                entities: [Post]
+            });
+            // Only for testing
+            (testOption.driver as any).schemaName = testSchema;
```

Comment:
why did you set schema name this name, and didnt pass schema name in connection options?